guacamole-docker/entrypoint.d/100-generate-guacamole-home.sh

```diff
@@ -59,8 +59,25 @@ is_property_set() {
 # Start with a fresh GUACAMOLE_HOME
 #
 
-rm -rf /tmp/guacamole-home.*
-GUACAMOLE_HOME="`mktemp -p /tmp -d guacamole-home.XXXXXXXXXX`"
+if [[ -z "$APACHE_BASE" ]]; then
```

Comment:
I'd request changing `APACHE_BASE` to `TOMCAT_BASE` or, perhaps, `WEBAPPS_BASE` throughout the code - Apache is the name of a large, non-profit software organization, so I'm not really sure it makes sense to use that as part of the variable.

Reply:
Thank you for the suggestion!